src/udev/udev-event.c

```diff
@@ -34,6 +34,11 @@
 #include "udev-watch.h"
 #include "user-util.h"
 
+#if HAVE_STAP
+#include <sys/sdt.h>
+#include "udev-probes.h"
+#endif
```

Comment:
hmm, thinking about this, wouldn't it be better, to move this into udev-util.h, given that that' where DEVICE_TRACE_POINT is defined, too?

Reply:
or taking this idea further, maybe add udev-trace-point.h for this? the macros is complex enough to deserve its own header file i think.

Reply:
Yeah we may want to add separate file in the future. Let's stick to `udev-util.h` for now.